webpack.config.js

```diff
@@ -1,21 +1,21 @@
-var path = require('path');
-var webpack = require('webpack');
-var postcssImport = require('postcss-import');
-var postcssNested = require('postcss-nested');
-var postcssCustomProps = require('postcss-custom-properties');
-var autoprefixer = require('autoprefixer');
-var postcssCalc = require('postcss-calc');
-var postcssColorFunction = require('postcss-color-function');
-var postcssMixins = require('postcss-mixins');
-var stylelint = require('stylelint');
-var WebpackGitHash = require('webpack-git-hash');
-var updateVersion = require('./updateVersion');
-var isDevelopment = 'development' === process.env.NODE_ENV;
+const path = require('path');
+const { DefinePlugin, HotModuleReplacementPlugin, optimize: { UglifyJsPlugin } } = require('webpack');
```

Comment:
This HAS to hit the 80 char limit, and as much as it's awesome, I think we should stick with the `webpack.DefinePlugin` format since that's what we do everywhere else.